okhttp/src/main/java/io/grpc/okhttp/Headers.java

```diff
@@ -45,8 +46,8 @@
    * creating a stream. Since this serializes the headers, this method should be called in the
    * application thread context.
    */
-  public static List<Header> createRequestHeaders(Metadata headers, String defaultPath,
-      String authority, String userAgent) {
+  public static List<Header> createRequestHeaders(
```

Comment:
Thanks for catching this. Removal from the public API surface was inadvertent, and now fixed. I added the new method with @ExperimentalApi, but did not pick an existing issue - https://github.com/grpc/grpc-java/issues/1775 is related but not quite right, so should probably create a new issue for this.